docs/source/chaincode_lifecycle.md

```diff
@@ -134,12 +134,13 @@ consistent across organizations:
 - **Name:** The name that applications will use when invoking the chaincode.
 - **Version:** A version number or value associated with a given chaincodes
   package. If you upgrade the chaincode binaries, you need to change your
-  chaincode version as well.
-- **Sequence:** The number of times the chaincode has been defined. This value
+  chaincode version as well. Versions may contain any characters,
+  but it is typical to use a format such as `v1.2.3`.
+- **Sequence:** The number of times the chaincode has been defined on a channel. This value
   is an integer, and is used to keep track of chaincode upgrades. For example,
-  when you first install and approve a chaincode definition, the sequence number
-  will be 1. When you next upgrade the chaincode, the sequence number will be
-  incremented to 2.
+  when you first approve and commit a chaincode definition on a channel, the sequence number
+  will be 1. When you next upgrade the chaincode or update the chaincode definition,
+  the sequence number will be incremented to 2.
```

Comment:
might be worth rephrasing to say that the 'user must increment it by 1 each time' 
make sure that people know then they need to up the number by 1 each time.. i.e. that the peer will not do it for you.